README.md

```diff
@@ -3,13 +3,13 @@ This is a TypeScript library for the Bitcoin ARC project.
 
 ## Installation
 ```bash
-npm install js-arc-client
+npm install @bitcoin-a/arc-client
 ```
 
 ## Usage
 
 ```javascript
-import { ArcClient } from 'js-arc-client';
+import { ArcClient } from '@bitcoin-a/arc-client';
```

Comment:
Let's move them to @bsvb - it's cleaner, and we're changing branch to bsv blockchain association - bitcoin-a doesn't really roll off the tongue or fingers.

Reply:
OK, yes that makes sense.
I'll close this PR then